README.md

```diff
@@ -1,2 +1,11 @@
 # browser-group-policies
-Group policies for different browsers to support companies to switch to Ecosia
+
+This repository includes configuration files that can be used to create Active Directory policies that will set Ecosia as the default search engine for different browsers.
```

Comment:
I copied that from the IE one. I don't really know how to clarify it (I still don't fully understand the whole windows admin ecosystem). Could you maybe give me some info you'd want to be in there to clarify it?